src/Microsoft.Health.Fhir.Core.UnitTests/Models/PartialDateTimeTests.cs

```diff
@@ -224,19 +174,11 @@ public void GivenADateImtOffset_WhenInitialized_ThenCorrectPartialDateTimeShould
             const int second = 59;
             const int millisecond = 153;
             const int fraction = 1234;
-            TimeSpan utcOffset = TimeSpan.FromMinutes(240);
+            var utcOffset = TimeSpan.FromMinutes(240);
 
-            var dateTimeOffset = new DateTimeOffset(
-                year,
-                month,
-                day,
-                hour,
-                minute,
-                second,
-                millisecond,
-                utcOffset).AddTicks(fraction);
+            DateTimeOffset dateTimeOffset = new DateTimeOffset(year, month, day, hour, minute, second, millisecond, utcOffset).AddTicks(fraction);
```

Comment:
Why change the others to vars but change this one to be explicitly typed?

Reply:
I believe I was trying to make ReSharper happy haha. I'll make my changes consistent across the file.

Reply:
The guidance is to use the explicit type when it isn't obvious, in this case you could use var since you directly see the type :) pretty sure Resharper has a setting to support this.

Reply:
Changed to var in 6c0373a.